scss/_patterns_aside.scss

```diff
@@ -0,0 +1,58 @@
+@import 'settings';
+
+@mixin vf-p-aside {
+  .p-aside {
+    align-self: flex-start;
+    border-top: 1px solid $color-mid-light;
+    flex: 0 0 16rem;
+    font-size: .875rem;
+    padding: 0 $sp-medium;
+
+    @media (min-width: $breakpoint-medium) {
+      border-left: 1px solid $color-mid-light;
+      border-top: 0;
+      margin-top: $sp-xxx-large * 2;
+      padding: 0 $sp-x-large;
+    }
+
+    &__header {
+      color: $color-mid-dark;
+      font-size: $sp-medium;
+      margin-bottom: $sp-x-small;
+      text-transform: uppercase;
+    }
+
+    &__section {
+      padding: $sp-large 0;
+
+      &:not(:last-child) {
+        border-bottom: 1px dotted $color-mid-light;
+      }
+    }
+
+    &__nav {
+      list-style: none;
+      margin: 0;
+      padding: 0;
+
+      .nav__link {
```

Comment:
Is this part of a separate pattern called `nav`?

Reply:
No - but to nest further within this pattern would create grandchild nesting - `.p-aside__nav__link`

Reply:
You shouldn't continue to nest elements with BEM. Instead, modify the name of the element. In this case, I would suggest something like `p-aside__nav-link` or simply `p-aside__link`.